frontend/packages/console-app/src/components/detect-namespace/namespace.ts

```diff
@@ -56,7 +65,19 @@ export const useValuesForNamespaceContext = () => {
     // Automatically sets favorited or latest namespace as soon as
     // both informations are loaded from user settings.
     if (!urlNamespace && favoriteLoaded && lastNamespaceLoaded) {
-      dispatch(setActiveNamespace(favoritedNamespace || lastNamespace || ALL_NAMESPACES_KEY));
+      const setActiveIfExists = (ns: string) => {
+        if (!ns) {
+          return Promise.reject();
+        }
+        return namespaceExists(useProjects ? ProjectModel : NamespaceModel, ns)
+          .then(() => dispatch(setActiveNamespace(ns)))
+          .catch((res) => {
+            // eslint-disable-next-line no-console
+            console.warn('Error fetching namespace', ns);
+            throw res;
+          });
+      };
+      setActiveIfExists(favoritedNamespace).catch(() => setActiveNamespace(lastNamespace));
```

Comment:
I assume the call inside the catch should be `setActiveIfExists`.

```suggestion
      setActiveIfExists(favoritedNamespace).catch(() => setActiveIfExists(lastNamespace));
```

I have a feeling we'll need a catch here to avoid a unhandled promise rejection warnings. So maybe it needs to be

```suggestion
      setActiveIfExists(favoritedNamespace).catch(() => setActiveIfExists(lastNamespace).catch(() => {});
```

Reply:
Linter dislikes the latter:  "Avoid nesting promises."

Reply:
I haven't tested, but I have a feeling the linter wants...

```suggestion
      setActiveIfExists(favoritedNamespace).catch(() => setActiveIfExists(lastNamespace)).catch(() => {});
```

Reply:
Yep.  That was it.  Thanks, @spadgett.  Fixed.